pkg/cli/config.go

```diff
@@ -208,6 +211,9 @@ func (c *Config) RegisterCommandLineFlags() {
 		flag.StringVar(&c.Backend.FileDir, "keyring-file-dir", keyringDirectory, "keyring `directory` for file-backed keyring types")
 		flag.BoolVar(&c.Debug, "keyring-debug", false, "Enable keyring debug logging")
 	}
+	if c.Flags.isSet(FlagBLE) && c.Flags.isSet(FlagVIN) {
+		flag.BoolVar(&c.BtTinyGo, "tinygo", false, "Use tinygo ble impl (go-ble is the default.")
```

Comment:
God help us if we end up needing more than two adapter implementations, but thoughts on bool vs string here?

Reply:
yes, will do